topology/probes/netlink.go

```diff
@@ -373,11 +373,14 @@ Loop:
 	for {
 		n, err := syscall.EpollWait(epfd, events[:], 1000)
 		if err != nil {
-			logging.GetLogger().Error("Failed to receive from netlink messages: %s", err.Error())
-			continue
+			errno, ok := err.(syscall.Errno)
+			if ok && errno != syscall.EINTR {
```

Comment:
I tested it after, yeah the go syntax is a little bit wierd like :+1: 
Loop:
   for {
    if x {
        break Loop  // looks like a goto, but it just to nominate a loop in go in case of multiple nested loop.
    } 
}